### DISCUSSION TITLE
Items Icone is empty!

### DISCUSSION THREAD

**Discussion by HotkaOriginal** [CustomiesDevs/Customies#124] [Q&A]
My items are just with empty texture! I will leave my plugin and plugin_data tomorrow here. And can someone leave here a plugin-example of custom item please?

**Comment by unickorn** [comment]
Hey! Have you checked out [the wiki](https://github.com/CustomiesDevs/Customies/wiki)? There are examples for custom blocks, entities and items there with code snippets, as well as a link to an example resource pack.

**Comment by and7ey** [comment]
@unickorn I have the same problem - textures are not shown for my custom item. The following error is shown
`Item item.alleva:blindness_sword requires either an icon atlas or icon texture.`

I tried another plugin - https://github.com/DavyCraft648/Oceanite - textures also are not shown in this case.

Not sure how to use your Wiki example, the RP is there, but is not according plugin.
